blocks/hero/hero.js

```diff
@@ -35,25 +35,31 @@ function buildHeroBlock(element) {
  * Decorates discount bubble div
  */
 function decorateDiscountBubble() {
-  // Example: <p><a href="example.com">Text</a> 50% Discount</p>
```

Comment:
An example like this might be good to be remain in the code

Reply:
I updated the code according to this example:
```
<p><a href="example.com">Text</a> <em>50% Discount</em></p>
```